docs/api.md

```diff
@@ -294,6 +294,13 @@ The fields are :
   objects. This can be used to limit the set of available `BareMetalHost`
   objects chosen for this `Machine`.
 
+* **automatedCleaningMode** -- An interface to enable or disable Ironic automated cleaning during provisioning
+  or deprovisioning of a host. When set to `disabled`, automated cleaning will be skipped, where
+  `metadata`(default) value enables it. It is recommended to tune the cleaning via metal3MachineTemplate rather than
+  metal3Machine. When `spec.template.spec.automatedCleaningMode` field of metal3MachineTemplate
+  is updated, metal3MachineTemplate controller will update all the metal3Machines (cloned from the metal3MachineTemplate)
```

Comment:
is `cloned` the proper terminology ?
If not, `generated from` metal3MachineTemplate ... would be more appropriate.

Reply:
done, thanks